python/lsst/dax/apdb/sql/apdbSqlReplica.py

```diff
@@ -94,6 +94,10 @@ def _timer(self, name: str, *, tags: Mapping[str, str | int] | None = None) -> T
         """Create `Timer` instance given its name."""
         return Timer(name, *self._timer_args, tags=tags)
 
+    def schemaVersion(self) -> VersionTuple:
+        # Docstring inherited from base class.
+        return self._schema.schemaVersion()
```

Comment:
And this will need more work, as `ApdbSql` does not remember schema version, I'd suggest to modify `ApdbSql`:
- Make `ApdbSql._versionCheck()` return `db_schema_version` and store it in `self._db_schema_version`.
- Add a parameter to `ApdbSqlReplica` that takes that version number.